src/backend/distributed/executor/repartition_logic.c

```diff
@@ -142,28 +138,7 @@ CreateSchemasOnAllWorkers(char *createSchemasCommand)
 {
 	List *commandList = list_make1(createSchemasCommand);
 
-	SendCommandToAllWorkers(commandList);
-}
-
-
-/*
- * SendCommandToAllWorkers sends the given command to all workers in
- * a single transaction.
- */
-static void
-SendCommandToAllWorkers(List *commandList)
-{
-	ListCell *workerNodeCell = NULL;
-	char *extensionOwner = CitusExtensionOwnerName();
-	List *workerNodeList = ActiveReadableNodeList();
-
-	foreach(workerNodeCell, workerNodeList)
-	{
-		WorkerNode *workerNode = (WorkerNode *) lfirst(workerNodeCell);
-		SendCommandListToWorkerInSingleTransaction(workerNode->workerName,
-												   workerNode->workerPort, extensionOwner,
-												   commandList);
-	}
+	SendCommandListToAllWorkers(commandList);
```

Comment:
This function sends commands as superuser, which is undesirable. We may want to add a user argument to it.

Reply:
Thinking about it more, sending this as superuser is useful because it allows us to revoke all access to the UDFs.